doc/source/getting_started.rst

```diff
@@ -0,0 +1,359 @@
+.. currentmodule:: larray
+
+Getting Started
+===============
+
+To use the LArray library, the first thing to do is to import it
+
+.. ipython:: python
+
+    from larray import *
+
+Create an array
+---------------
+
+Working with the LArray library mainly consists of manipulating :ref:`LArray <api-larray>` data structures.
+They represent N-dimensional labelled arrays and are composed of data (numpy ndarray), :ref:`axes <api-axis>`
+and optionally a title. An axis contains a list of labels and may have a name (if not given, the axis is anonymous).
+
+You can create an array from scratch by supplying data, axes and optionally a title:
+
+.. ipython:: python
+
+    # define data
+    data = [[20, 22], [33, 31], [79, 81], [28, 34]]
+
+    # define axes
+    age_category = Axis(["0-9", "10-17", "18-66", "67+"], "age_category")
+    sex = Axis(["M", "F"], "sex")
+
+    # create LArray object
+    arr = LArray(data, [age_category, sex], "population by age category and sex")
+    arr
+
+.. note::
+
+   LArray offers two syntaxes to build axes and making selections and aggregations.
+   The first one follows the Python syntax and allow to use any characters in labels.
```

Comment:
* s/follows the Python syntax/is more Pythonic (uses Python structures)/ (both follow the Python syntax since we do not get SyntaxError)
* s/allow/allows/
* s/characters/character/